src/user/announcements.rst

```diff
@@ -5,6 +5,12 @@ Announcements
 ----
 
 
+:2020-07-15: Moving from clang 9 to clang 10
```

Comment:
I'm not sure how ReST works, should we have only a single `2020-07-15` header? If so just offsetting by the same amount of spaces the CFEP-18 would work?

Reply:
I like things that solves themselves by just waiting a little bit :smile: